.travis.yml

```diff
@@ -26,12 +26,6 @@ before_install:
       sleep 3;
     fi
 
-  - cd ./vscode-client
```

Comment:
Please do not import the changes about vscode in this PR since we have a separate PR to do it :smile:

Reply:
This PR builds on top of #7 with https://github.com/coala/coala-ls/pull/2/commits/cbbaaf89d43e9b3800c633ea06c99fa5f21c95d8. These changes are a part of that commit. CI will fail if not for this change.

Reply:
OK, then we could remove it after #7 merged.